packages/reakit/src/Dialog/__utils/useFocusTrap.ts

```diff
@@ -112,21 +114,24 @@ export function useFocusTrap(
   // Click trap
   React.useEffect(() => {
     if (!shouldTrap) return undefined;
+    const doc = portalRef.current?.ownerDocument || document;
```

Comment:
Probably this is never the case but if the ref is `null`, doc's assigned the wrong `document` right?

Reply:
Should `||` be `??` here?

Reply:
Should `||` be `??` here?

Reply:
That may be the case if the user doesn't pass `ref` to the component. We can probably show a warning there.

Reply:
That `||` is there more because TypeScript said that `ownerDocument` may be `null`. Haven't research yet in which cases that would be true.

Reply:
According to MDN, it seems that it would be `null` if the node (`portalRef.current` in that case) is the document itself.